content/applications/hr/payroll/contracts.rst

```diff
@@ -2,224 +2,372 @@
+To view the employee contracts, go to :menuselection:`Payroll app --> Contracts --> Contracts` from
+the top menu. All employee contracts are displayed in a default list view, grouped into four
+categories: :guilabel:`New`, :guilabel:`Running`, :guilabel:`Expired`, and :guilabel:`Cancelled`.
+
+The default view has all categories collapsed. The number of contracts in each category is displayed
+next to the category name (example: `Running (20)`). To view the list of contracts under any
+category, click on the category, and the list expands downward, showing all contracts beneath it.
 
 .. image:: contracts/contracts-overview.png
    :align: center
    :alt: Contracts dashboard view showing running contracts and contracts with issues.
 
 .. note::
    The list of contracts in the *Payroll* application matches the list of contracts in the
-   *Employees* application. The default contracts view in the *Payroll* application displays running
-   contracts and contracts needing attention, while the default contracts view in the *Employees*
-   application displays all contracts in a kanban view, organized by their stage, regardless of
-   status. All contracts can be viewed by changing the filters.
-
-Create a new contract
-=====================
+   *Employees* application.
+
+.. _payroll/new-contract:
+
+Create new contracts
+====================
+
+For an employee to be paid, an active contract is required. When a new contract is needed, click the
+:guilabel:`New` button on the :guilabel:`Contracts` dashboard. A contract form appears where the
+information can be entered.
+
+.. _payroll/gen-info:
+
+General information section
+---------------------------
+
+- :guilabel:`Contact Reference`: type in the name or title for the contract, such as `John Smith
+  Contract`. This field is **required**.
+- :guilabel:`Employee`: using the drop-down menu, select the employee that the contract applies to.
+- :guilabel:`Contract Start Date`: the date the contract starts. To choose a date, click the
+  drop-down menu, navigate to the correct month and year with the :guilabel:`< > (arrow)` icons,
+  then click on the desired date. This field is **required**.
+- :guilabel:`Contract End Date`: if the contract has a specific end date, click the drop-down menu,
+  navigate to the correct month and year with the :guilabel:`< > (arrow)` icons, then click on the
+  desired date.
+- :guilabel:`Working Schedule`: select one of the working schedules from the drop-down menu. This
+  field is **required**.
+
+  .. tip::
+     The :guilabel:`Working Schedule` drop-down menu displays all the working schedules for the
+     selected company. To modify or add to this list, go to :menuselection:`Payroll app -->
+     Configuration --> Working Schedules`. Click :guilabel:`New`, and create a new working schedule,
+     or click on an existing working schedule and make edits.
+
+- :guilabel:`Work Entry Source`: select how the :doc:`work entries </work_entries>` are generated.
```

Comment:
@larm-odoo Build error showing the an issue with the path https://runbot.odoo.com/runbot/build/60855632

No need for the `/` before `work_entries`, this would mean that `work_entries.rst`is at the top-level `content/work_entries.rst` location (which it isnt) :)
```suggestion
- :guilabel:`Work Entry Source`: select how the :doc:`work entries <work_entries>` are generated.
```